src/common/options.cc

```diff
@@ -7225,6 +7225,10 @@ static std::vector<Option> get_rbd_options() {
     .set_default(true)
     .set_description("validate new image names for RBD compatibility"),
 
+    Option("rbd_validate_object_map", Option::TYPE_BOOL, Option::LEVEL_DEV)
```

Comment:
Nit: perhaps this should be `rbd_invalidate_object_map_on_load_timeout`. It's not "validating" the object map and it's only for that one specific case.